pkg/smokescreen/acl/v1/acl.go

```diff
@@ -83,7 +83,7 @@ func (acl *ACL) Add(svc string, r Rule) error {
 		return err
 	}
 
-	err = acl.ValidateRuleDomainsGlobs(svc, r)
+	err = acl.ValidateRuleDomainsGlobsAndMitm(svc, r)
```

Comment:
Should this actually just be `ValidateRule` since we're passing in the whole rule now?

Reply:
Good idea!